fabric-content-registries-v0/build.gradle

```diff
@@ -1,5 +1,9 @@
 archivesBaseName = "fabric-content-registries-v0"
-version = getSubprojectVersion(project, "0.2.2")
+version = getSubprojectVersion(project, "0.2.3")
+
+loom {
+	accessWidener = file("src/main/resources/fabric-content-registries-v0.accesswidener")
```

Comment:
This should be a mixin. Using an AW should only be done when required. Especially in large multi project builds like fab api as it will slow things down.

Reply:
...and the requirement is right there in the PR description:

> (I had to use an AW instead of an accessor due to SpongePowered/Mixin#399 - the two addFuel overloads broke the Mixin AP.)

Reply:
Ah my bad. This is fine then.